R/formatter.r

```diff
@@ -1,6 +1,16 @@
-#' Number formatter: a generic formatter for numbers
+#' Number formatters
 #'
-#' @return `number_format` returns a function with single parameter
+#' `number_format()` and `number()` are generic formatters for numbers.\cr
```

Comment:
Please eliminate the `\cr` - these should all run into together in one paragraph. (Or if you don't like that, use a bulleted list instead)

Reply:
Or maybe you want to use `@description` so you can use regular new lines?

Reply:
done